src/components/Card/Card.module.scss

```diff
@@ -5,19 +5,20 @@
   border: 1px solid $cardBorder;
   border-radius: 1em;
   margin-bottom: 2em;
-  max-width: 850px;
+  max-width: 950px;
 
   &__img {
```

Comment:
Can't be image - because you have image inside. It's image container or smth like that

Reply:
Please clarify, I don't understand

Reply:
You mean container name __img not accurate ?